Sources/UtilityBeltUITesting/Extensions/StubbedDataCollection+LaunchEnvironmentCodable.swift

```diff
@@ -0,0 +1,9 @@
+// Copyright © 2021 SpotHero, Inc. All rights reserved.
+
+import Foundation
+import Sham
+
+extension StubbedDataCollection: LaunchEnvironmentCodable {
+    /// The key used to save and retrieve the object in the launch environment.
+    public static var launchEnvironmentKey = "mocked-data"
```

Comment:
any thoughts to making this an enum and then require the `LaunchEnvironmentCodable` to have a value of that type?  That would have the added benefit of ensuring string keys are not duped and would also keep all of those declarations in one spot.

Reply:
That's a good idea. Currently, there is a card to handle collisions, https://spothero.atlassian.net/browse/IOS-2690. For now, would you be ok with me leaving this PR as-is and implementing this idea when that card comes around?

Reply:
Although, I don't think this stops two different objects with the same `launchEnvironmentKey` enum value being sent to the launch environment coding function.

Reply:
ya this would only stop duplicate raw values.  It was just a thought on my end, im totally fine with doing it in the other ticket.